src/app/requirements.txt

```diff
@@ -52,14 +52,10 @@ idna==3.2
     # via requests
 imagesize==1.2.0
     # via sphinx
-importlib-metadata==4.6.4
```

Comment:
It seems unexpected that some things got removed form the requirements.txt when you added the isort requirement.  This seems to stem from the fact that you used python 3.8 to generate this file.  On python 3.7,  these required packages stay.  Since 3.7 is the minimum supported version, we should probably always generate this file using that version.